src/automations/index.ts

```diff
@@ -50,6 +71,6 @@ export const automations = (
 ): ICourierClientAutomations => {
   return {
     invokeAdHocAutomation: invokeAdHocAutomation(options),
-    invokeAutomationTemplate: invokeAutomationTemplate(options)
+    invokeAutomationTemplate: invokeAutomationTemplate(options),
```

Comment:
`yarn format` specified in `Contribution.md` won't like comma at the end - can you check what `yarn format` does to your change?